app/src/main/java/org/fossasia/openevent/app/module/auth/signup/SignUpFragment.java

```diff
@@ -68,6 +68,12 @@ public void onStart() {
                 return;
 
             String url = binding.url.baseUrl.getText().toString().trim();
+            String password = binding.password.getText().toString();
+            String confirmPassword = binding.confirmPassword.getText().toString();
+            if (!(getPresenter().comparePassword(password, confirmPassword))) {
```

Comment:
This translates to `if passwords are not compared, return` 

Functions with verbs as names return void, so this doesn't make sense

Functions which return boolean should be named as `arePasswordsEqual`

Reply:
I apologize . I'll do the required changes.